ext/standaloneusers/ang/crmChangePassword/crmChangePassword.html

```diff
@@ -1,50 +1,61 @@
 <div>
-  <h1>Forgotten Password</h1>
-  <form name="changePassword" crm-ui-id-scope>
+  <form name="changePassword" crm-ui-id-scope ng-submit="$ctrl.attemptChange()">
 
     <div crm-ui-field="{name: 'actorPassword', title: ts('Enter your current password')}">
-      <input
-        crm-ui-id="actorPassword"
-        name="actorPassword"
-        ng-model="$ctrl.actorPassword"
-        class="crm-form-text"
-        type=password
-      />
+      <input crm-ui-id="actorPassword" name="actorPassword" ng-model="$ctrl.actorPassword" class="crm-form-text"
+        type=password />
     </div>
 
     <div crm-ui-field="{name: 'newPassword', title: ts('Enter a new password')}">
-      <input
-        crm-ui-id="newPassword"
-        name="newPassword"
-        ng-model="$ctrl.newPassword"
-        class="crm-form-text"
-        type=password
-      />
+      <input crm-ui-id="newPassword" minlength=11 name="newPassword" ng-model="$ctrl.newPassword" class="crm-form-text"
+        type=password />
```

Comment:
I think adding `autocomplete="new-password"` to this and the newPasswordAgain input allows password managers to latch on to these (if I've understood https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete correctly)